validation/contextual-services/quicksuggest-impression.1.event.pass.json

```diff
@@ -0,0 +1,20 @@
+{
+  "context_id": "{94642acb-4996-034b-916c-147da723cc41}",
+  "search_query": "Puma running shoes",
+  "matched_keywords": "Puma shoes",
+  "is_clicked": false,
+  "advertiser": "amazon",
```

Comment:
I don't think it really matters but I would prefer not to use real brands for test values

Reply:
Good point, we should do that.